metadata-ingestion/src/datahub/ingestion/extractor/json_schema_util.py

```diff
@@ -275,6 +275,10 @@ def _field_from_primitive(
     @staticmethod
     def _get_type_from_schema(schema: Dict) -> str:
         """Returns a generic json type from a schema."""
+        # Handle boolean schemas per JSON Schema spec: true accepts any JSON, false never validates
```

Comment:
is there any public reference to the json schema spec that we could add here as comment?